src/main/scala/is/hail/methods/LDMatrix.scala

```diff
@@ -40,12 +41,30 @@ object LDMatrix {
     normalizedBlockMatrix.blocks.count()
     normalizedIRM.rows.unpersist()
 
-    val indexedRowMatrix = (normalizedBlockMatrix multiply normalizedBlockMatrix.transpose)
-      .toIndexedRowMatrix()
+    val thresholdSize = 5000 * 5000
+    val numEntries: Long = normalizedBlockMatrix.numCols() * normalizedBlockMatrix.numRows()
+
+    if (optComputeLocally == null) {info("optComputeLocally is null")}
```

Comment:
Oh I'm sorry this should not have been there. This was a debugging statement I forgot to remove.